tiledb/ml/data_apis/tensorflow_sparse.py

```diff
@@ -75,6 +81,37 @@ def __new__(cls, x_array: tiledb.Array, y_array: tiledb.Array, batch_size: int):
                 ),
             )
 
+    @classmethod
+    def __check_row_dims(
```

Comment:
I guess you will also need the same unit test as in tf.

Reply:
Correct! Added them!